### PR TITLE
Update spec file to match production

### PR DESCRIPTION
**Pull request by smarlowucf**

### CODE REVIEW

**Comment by rjschwei** [comment]
Hmm, this change implies that in a project where SLES 12 builds are enabled we will build with python2, which from my perspective we do not want. Alternatively it implies that this package does not inherit the project config and we disable builds in those projects for SLES 12 specifically for this package. I fail to see the advantage of the changes.

File: python3-aliyun-img-utils.spec [hunk 1]
```diff
@@ -15,31 +15,38 @@
 # Please submit bugfixes or comments via https://bugs.opensuse.org/
 #
 
-
-Name:           python3-aliyun-img-utils
+%{?!python_module:%define python_module() python-%{**} python3-%{**}}
+%if 0%{?suse_version} >= 1500
+%define skip_python2 1
```
Comment (rjschwei) [inline python3-aliyun-img-utils.spec:20]: Python2 builds should be skipped. I don't think we build the dependencies as python2 either.

Comment (smarlowucf) [reply python3-aliyun-img-utils.spec:20]: This is not a new change. The goal with this PR was to sync the spec file from the build service.

Comment (rjschwei) [reply python3-aliyun-img-utils.spec:20]: OK, let's get the conflicts resolved and get it merged then.

**Pull request merged**